proto/one/instrument/twin/external/key_value_claros.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+option csharp_namespace ="ONE.Models.CSharp.External";
+
+message KeyValueClaros {
```

Comment:
I might suggest making `Claros` the prefix so the differentiation is more obvious.  If I start typing `KeyValue` intellisense could give me `KeyValue` or `KeyValueClaros` but it probably would not give me `ClarosKeyValue` does that make sense?